### ISSUE TITLE
convergence of training process

### ISSUE THREAD

**Issue by allen-ash** [duxingren14/DualGAN#18]
In the DualGAN paper shows Algorithm 1 DualGAN training procedure.  It says stop training until convergence. Do you have any idea what exactly it means? Does it means A_d_loss and B_d_loss update to 0.5 Simultaneously？Many thanks.

**Comment by duxingren14** [comment]
No, convergence means the image quality reaches satisfactory. Practically, you need to specify the args.epoch manually by observing the image quality.
